imports/ui/JapaneseNewGame.js

```diff
@@ -790,7 +791,7 @@ function save_game_to_database(hands_array) {
 		if (Number(Session.get("west_score")) >= Number(Session.get("north_score"))) position--;
 		Players.update({_id: west_id}, {$inc: {japanesePositionSum: position}});
 
-		switch (position) {
+    switch (position) {
```

Comment:
There's generally a lot of ruined indentation in this patch.  Please fix.  (Tabs for indentation, spaces for alignment)